src/rfxencode_diff_rlgr1.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol client.
+ * RemoteFX Codec Library - RLGR
+ *
+ * Copyright 2011 Vic Lee
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __RFX_DIFF_RLGR1_H
+#define __RFX_DIFF_RLGR1_H
+
+#include "rfxcommon.h"
+
+int
+rfx_encode_diff_rlgr1(sint16* coef, uint8* cdata, int cdata_size);
```

Comment:
Please format newly added code. The asterisks are on the wrong side.

If any pointers point to constant data, please say use `const`.

The functions should be documented somewhere. Perhaps a separate markdown file would be the best.